device/iot-device-client/pom.xml

```diff
@@ -82,6 +83,11 @@
             <artifactId>commons-codec</artifactId>
             <version>1.10</version>
         </dependency>
+        <!-- Logging support -->
+        <dependency>
+   			<groupId>org.slf4j</groupId>
+ 			<artifactId>slf4j-api</artifactId>
+    	</dependency>
```

Comment:
I have done this in a separate PR #57 . I tried to follow the three steps described above as good as possible.